src/components/HomePage/billboardCarousel.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { BillboardContentBox, CarouselInner } from "./homePage.styled";
+import { Typography } from "@mui/material";
+
+interface homeProps {
+  images?: any;
+  currentBillboard?: any;
+}
+
+export const BillboardCarousel: React.FunctionComponent<homeProps> = (
+  props
+) => {
+  const { images, currentBillboard } = props;
+
+  return (
+    <>
+      {images.map((image: any, index: 0) => (
```

Comment:
let's try to avoid any, 

Image in this place should look like the object being passed in.

And besides I don't see where you are using the `image` variable

Index isn't type `0`, it is type `number`